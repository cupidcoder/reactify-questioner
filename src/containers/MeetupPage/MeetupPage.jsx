import React from 'react';
import BaseLayout from '../BaseContainer/BaseContainer';
import MeetupDetailCard from '../../components/MeetupDetailCard/MeetupDetailCard';
import RsvpButtons from '../../components/RsvpButtons/RsvpButtons';
import style from './MeetupPage.css';

const MeetupPage = () => {
  const meetupDetail = {
    name: 'The Power of X',
    location: 'Gbagada',
    date: '31st of May, 2019',
    description: `To unpack what this means, how to hop on the remote train,
    and be successful at it, Andela is putting together this evening event with
    Nigeria's Tech Leaders, Andelans, and Tech Expert from an amazing company who we're stoked to work with!.`,
    attending: 189
  };
  return (
    <BaseLayout>
      <div className={style.MeetupPage}>
        <MeetupDetailCard {...meetupDetail} />
        <RsvpButtons />
      </div>
    </BaseLayout>
  );
};

export default MeetupPage;
