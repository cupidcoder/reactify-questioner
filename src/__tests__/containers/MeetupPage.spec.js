import React from 'react';
import { shallow } from 'enzyme';
import { MeetupPage } from '../../containers/MeetupPage/MeetupPage';

describe('MeetupPage', () => {
  it('Should correctly save snapshot of the MeetupPage', () => {
    const wrapper = shallow(<MeetupPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
