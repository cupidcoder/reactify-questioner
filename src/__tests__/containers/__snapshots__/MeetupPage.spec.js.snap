// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MeetupPage Should correctly save snapshot of the MeetupPage 1`] = `ShallowWrapper {}`;
